include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
Looks like you run vcbuild.bat which used Visudl Studio 2010 and then tried to use it from 2015. Nodejs does not run into any of these problems...